functions/helloworld/hello-http/src/test/java/functions/ExampleSystemIT.java

```diff
@@ -34,6 +35,10 @@
   // TODO<developer>: set this value, as an environment variable or within your test code
   private static final String BASE_URL = System.getenv("FUNCTIONS_BASE_URL");
 
+  // Access token used to send requests to authenticated-only functions
+  // TODO<developer>: set this value if your function requires authentication
```

Comment:
You might wish to explain how, or tell users where to find out how.

Reply:
Added a docs link.